.github/workflows/CodeQuality.yaml

```
@@ -67,3 +61,14 @@ jobs:
         run: docker compose build
       - name: Run linters
         run: docker compose run --rm app npm run lint
+
+sonarqube:
```

actionlint (1.7.4)

unexpected key "sonarqube" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)